### ISSUE TITLE
Change main click

### ISSUE THREAD

**Issue by Ichiro91** [hwnd-git/Diablo-IV-AHK-WASD#7]
Hello ! any way to change the main left click to something else like right click please ?

**Comment by ragestaker21** [comment]
i want to know this

**Comment by hwnd-git** [comment]
It doesn't work for me, but as far as I remember from YT comments, it works for some people. I don't know the reason why such inconsistency. Try it by yourself by changing "L" to "R" in all commands in the code: "ControlClick, x%xCenter% y%yCenter%, A,, L, 1, NA" -> "ControlClick, x%xCenter% y%yCenter%, A,, R, 1, NA"

**Comment by ragestaker21** [comment]
yes i already tried it doesnt work its like this script doesnt work even if you change mouse clicks

**Comment by ragestaker21** [comment]
i m trying to use it for league of legends, also changed appid correctly i also hear sounds but it doesnt work.

for diablo 4 the default settings works very good for me.
